public/app/features/dimensions/editors/FileUploader.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { FileDropzone, useTheme2, Button, DropzoneFile } from '@grafana/ui';
+import { getBackendSrv, config } from '@grafana/runtime';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { MediaType } from '../types';
+interface Props {
+  setNewValue: Dispatch<SetStateAction<string>>;
+  setFormData: Dispatch<SetStateAction<FormData>>;
+  mediaType: MediaType;
+  setUpload: Dispatch<SetStateAction<boolean>>;
+  getRequest: (formData: FormData) => Promise<UploadResponse>;
+}
+interface UploadResponse {
+  err: boolean;
+  path: string;
+}
+export function FileDropzoneCustomChildren({ secondaryText = 'Drag and drop here or browse' }) {
+  const theme = useTheme2();
+  const styles = getStyles(theme);
+
+  return (
+    <div className={styles.iconWrapper}>
+      <small className={styles.small}>{secondaryText}</small>
+      <Button icon="upload">Upload</Button>
```

Comment:
```suggestion
      <Button type="button" icon="upload">Upload</Button>
```
By default the button type is `submit` and this can have unintended consequences when used inside a form.